### DISCUSSION TITLE
How to pass an nft_object as part of a transaction payload

### DISCUSSION THREAD

**Discussion by Pelumi527** [aptos-labs/aptos-developer-discussions#100] [Questions] [answered]
```
public entry fun add_token_v2_listing(
        signer_ref: &signer,   /// the seller
        nft_object: Object<TokenV2>,
        price: u64,
    ) 
```
I am trying to call this function from the frontend, but I am trying to get the` nft_object` for a V2 token to pass as part of the payload for the transaction

**Comment by kshitijc1506** [comment, marked as answer]
Using the SDK, you can just pass through the object's address. Here's an [example](https://github.com/aptos-labs/marketplace-example/blob/8b0d527ea0eed3db035110531ccf12889ed155cb/frontend/src/utils/aptos.ts#L124) for reference.
